SqlDataCatalog/Descope empty tables.ps1

```diff
@@ -21,5 +23,7 @@ $emptyTableColumnsInDboSchema = $emptyTableColumns | Where-Object { $_.schemaNam
 
 # Add-ClassificationColumnTag does a non-destructive update on all these columns (i.e. other tags are not removed)
 $emptyTableColumnsInDboSchema | Add-ClassificationColumnTag -category "Sensitivity" -tags "General"
-$emptyTableColumnsInDboSchema | Add-ClassificationColumnTag -category "Classification Scope" `
-    -tags "Out of Scope - Unused"
+$emptyTableColumnsInDboSchema | Add-ClassificationColumnTag -category $scopeCategory -tags $unusedTag
+
+$tablesNoLongerEmpty = $allColumns | Where-Object {($_.tags.Name -eq $unusedTag) -and ($_.tableRowCount -ne 0)}
+$tablesNoLongerEmpty | Set-ClassificationColumnTag -category $scopeCategory -tags @()
```

Comment:
Does this remove all of the tags rather than just the unused ones? Should it be only removing that one tag?

Reply:
Set-ClassificationColumnTag updates a single category.  So if `$scopeCategory` can have multiple tags assigned it will remove all of them.
In the default taxonomy `Classification Scope` is a single tag category, so for most users getting started this will work well.  
There isn't a commandlet to remove a single tag from a multi tag category at the moment